### DISCUSSION TITLE
PnP Provision Template Site title missing??? *HELP*

### DISCUSSION THREAD

**Discussion by LeonArmston** [pnp/powershell#2370] [Q&A]
Converting this to an issue as seems to be a problem between versions of PnP.PowerShell

### Discussed in https://github.com/pnp/powershell/discussions/2121

<div type='discussions-op-text'>

<sup>Originally posted by **cstaylor80** July  9, 2022</sup>
Hi I'm going out of brain at the moment. I have installed the latest version from pnp to powershell which i wanted to clone a team's site (group and non group enabled) which works fine. I then go to apply this template to another site with all the settings and while it does do it.. the destination site title in the header is missing?????. I have tried everything and exhausted google and have come up with nothing to make this work. The title is there in the site information however its NOT displayed on the home page of the site? why?

As mentioned I have uninstalled via PowerShell PnP and installed the current release and still the same problem?. is there something I am missing here? 

any help would be great as I need to find a way to display the current site name which has disappeared. 

![Screenshot 2022-07-09 102416](https://user-images.githubusercontent.com/24868053/178087119-83a46c44-123f-4f4c-95ff-341991b558cc.png)</div>

**Comment by veronicageek** [comment]
@LeonArmston - Can you please open an issue using the template so we keep things clean. I'm converting this back to a discussion and you can add your (new) issue as a reference for the wider audience. Thanks
